tests/unit_tests/plugins/module/output_checkers/test_output_checkers.py

```diff
@@ -0,0 +1,93 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test all Recenter TC Output Checker plugins."""
+import pytest
+import numpy as np
+from os import environ
+
+from geoips.interfaces import output_checkers
+from geoips.commandline import log_setup
+
+
+log_setup.setup_logging()
+
+
+class TestRecenterTCOutputCheckers:
+    """TestRecenterTCOutputChecker class, defining methods as well."""
+
+    savedir = str(environ["GEOIPS_PACKAGES_DIR"]) + "/test_data/test_fdecks/pytest/"
```

Comment:
@jsolbrig do we want to avoid Geoips packages dir? Should this just be relative to the pytest directory?

Reply:
I would like to avoid reference to `GEOIPS_PACKAGES_DIR`, but I don't really see a way to do that here right now. Unless we put the test files directly into the GeoIPS package.

Where do these test files currently come from? Are they big? Can they be included in GeoIPS directly?

Reply:
@jsolbrig I think largely this branch needs to be updated for the changes in geoips. Evan removed GEOIPS_PACKAEGS_DIR here in geoips: https://github.com/NRLMMD-GEOIPS/geoips/pull/379/files#r1374489732

I still think $GEOIPS_OUTDIRS/scratch/unit_tests or something could be appropriate for these auto-generated files.  That's the one place the user selects where it is expected files will go.

Reply:
This has been resolved